requirements.txt

```
@@ -1,4 +1,5 @@
-jsonschema
+jsonschema>=4.5; python_version >= '3.7'
+jsonschema; python_version < '3.7'
```

I get why you did this, but if the version checking works properly, it shouldn't be necessary to force the version instead of suggesting the user update
```suggestion
jsonschema
```